skbio/embedding/_embedding.py

```diff
@@ -7,10 +7,28 @@
 # ----------------------------------------------------------------------------
```

Comment:
Following the naming convention in scikit-bio, the name of this script file can be changed from `_embedding.py` into `_base.py`.

Reply:
Not necessarily. There is no `_base.py` in the stats folder for instance.